test/test.googleauth.ts

```diff
@@ -117,6 +118,8 @@ describe('googleauth', () => {
       .withArgs(wellKnownPathWindows)
       .returnsArg(0);
 
+    sandbox.stub(child_process, 'exec').callsArgWith(1, null, '', null);
```

Comment:
Can we scope this to only mock it with known parameters?  I just want to make sure we're being super explicit about what we want to mock.

```js
sandbox.stub(child_process, 'exec')
  .callThrough()
  .withArgs('gcloud config config-helper --format json')
  .callsArgWith(1, null, '');
```

Reply:
Done! `sinon` has some weird issues with types (it was forcing me to pass at least 2 parameters to `.withArgs` in this case), so I added this `undefined`. By looking at `.d.ts` I couldn't understand what went wrong.